backend/src/schema/types/type/REPORTED.gql

```diff
@@ -0,0 +1,42 @@
+type REPORTED {
+  createdAt: String
+  reasonCategory: ReasonCategory
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T08:28:02Z" title="Monday, October 14th 2019, 10:28:02 am +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,32 @@
+type REPORTED {
+  createdAt: String
+  reasonCategory: String
```


sorry, I thought we agreed this would be an enum of selected categories we provide in line with our Terms of Conditions, no?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-14T15:18:04Z" title="Monday, October 14th 2019, 5:18:04 pm +02:00">Oct 14, 2019</time>_

---


Yes, I forgot and did it locally yet. Push it up all together …